scala/Charts/src/main/scala/Taxi.scala

```diff
@@ -0,0 +1,38 @@
+import com.databricks.connect.DatabricksSession
+import org.apache.spark.sql.SparkSession
+import org.jfree.chart.{ChartFactory, ChartFrame, ChartUtils}
+import org.jfree.data.category.DefaultCategoryDataset
+import java.io.File
+
+object Taxi {
+  def createSession(): SparkSession = {
+    DatabricksSession.builder.remote().getOrCreate()
+  }
+
+  def main(args: Array[String]): Unit = {
+    val spark = createSession()
+
+    val data = spark.sql("""
```

Comment:
Could you add 1-2 more charts (other types)?

Would also be good to show one example of using dataframe APIs (instead of SQL)?

Reply:
Done